app/Resources/views/blog/post_show.html.twig

```diff
@@ -5,6 +5,13 @@
 {% block main %}
     <h1>{{ post.title }}</h1>
 
+    <p>
+        {% if 0 < post.tags | length %}Tags: {% endif %}
```

Comment:
I think should put the `if` block around the paragraph to avoid rendering an empty one (the same can be done above).